utils/test-utils/src/main/scala/pl/touk/nussknacker/test/NuRestAssureMatchers.scala

```diff
@@ -0,0 +1,170 @@
+package pl.touk.nussknacker.test
+
+import cats.implicits._
+import cats.Show
+import cats.data.Validated
+import com.typesafe.scalalogging.LazyLogging
+import io.restassured.response.ValidatableResponse
+import org.hamcrest.{BaseMatcher, Description, Matcher}
+import pl.touk.nussknacker.test.NuRestAssureMatchers.{EqualsJson, MatchJsonWithRegexValues}
+import ujson._
+
+import scala.util.Try
+
+trait NuRestAssureMatchers {
+  def equalsJson(expectedJsonString: String): Matcher[ValidatableResponse] =
+    new EqualsJson(expectedJsonString)
+
+  def matchJsonWithRegexValues(expectedJsonWithRegexValuesString: String): Matcher[ValidatableResponse] =
+    new MatchJsonWithRegexValues(expectedJsonWithRegexValuesString)
+}
+object NuRestAssureMatchers {
+
+  private type JSON = ujson.Value.Value
+
+  private class EqualsJson(expectedJsonString: String) extends BaseMatcher[ValidatableResponse] {
+
+    private val expectedJson = ujson.read(expectedJsonString)
+
+    override def matches(actual: Any): Boolean = {
+      actual match {
+        case str: String => ujson.read(str) == expectedJson
+        case _ => false
+      }
+    }
+
+    override def describeTo(description: Description): Unit = {
+      description.appendValue(expectedJsonString)
+    }
+  }
+
+  private class MatchJsonWithRegexValues(expectedJsonWithRegexValuesString: String)
```

Comment:
Nothing ready is available for all of this things? I see a lot of potential code that can has bugs. Is it possible to at least use some ready json traverse and use monocle to check only nested regexes?

Reply:
Too complicated to trus it works - I :heart: it ;)